generators/client/templates/angularjs/src/main/webapp/app/components/util/_data-util.service.js

```diff
@@ -77,7 +77,13 @@
         }
 
         function openFile (type, data) {
-            $window.open('data:' + type + ';base64,' + data, '_blank', 'height=300,width=400');
+            var fileURL = 'data:' + contentType +';base64,' + data;
```

Comment:
`contentType` is wrong it should be `type`

Reply:
Oops, I did not check damn. You are right. @NilsWild can you please fix it

Reply:
how did this pass linting :thinking: